02_activities/assignments/assignment.sh

```diff
@@ -8,17 +8,22 @@ set -x
 # > Add your code below each comment to complete the tasks
 
 # 1. Create 5 directories named dir1, dir2, dir3, dir4, and dir5
-
+mkdir dir1 dir2 dir3 dir4 dir5
 # 2. List the contents of the current working directory to verify the presence of the 5 directories
-
+ls
 # 3. Create 5 text files in dir2 named file1.txt, file2.txt, file3.txt, file4.txt, and file5.txt
-
+cd dir2
+touch file1.txt  file2.txt file3.txt file4.txt file5.txt
 # 4. Append the words "Hello world" to dir2/file3.txt
-
+echo "Hello world" >> dir2/file3.txt
 # 5. Verify that file3.txt contains the words "hello world" by printing the contents of the file in the terminal
-
+cd dir2
+ls
+cat file3.txt
 # 6. Delete file4.txt
-
+cd dir2
```

Comment:
This is a general comment to your code:

For better practice, specify dir2 within your file path so you don't have to cd into dir2 everytime you want to modify a file.